server/controllers/request.js

```diff
@@ -0,0 +1,99 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+/*
+@route GET /requests/getRequests
+@desc for fetching all the requests for a particular user
+@access private
+*/
+exports.getRequests = asyncHandler( async( req, res, next ) => {
+    const  user_id  = req.query.user_id;
+    
+    const user = await Request.find({ user_id: user_id }, function(err) {
```

Comment:
Maybe rename this variable to requests (here and below), to increase semantic meaning?